src/Blog/BlogPlaceholder.scss

```diff
@@ -0,0 +1,123 @@
+@import "../styles/typography";
+.blog-section {
+  background-image: url("../../assets/blogbackground.jpg");
+  opacity: 0.75;
+  background-size: cover;
+  height: 100vh;
```

Comment:
I think this should be calc(100vh - 60px), it goes over the height of the page for me